app/main.py

```diff
@@ -1 +1,10 @@
-# write your code here
+def copy_file(string: str) -> None:
+    names = string.split(" ")
+    if len(names) >= 3 and names[0] == "cp":
```

Comment:
Do not perform the check for command in this if condition, devote it to only length checking.